src/v3/src/constants/idxConstants.ts

```diff
@@ -161,13 +155,6 @@ export const STEPS_REQUIRING_CUSTOM_LINK = [
   IDX_STEP.AUTHENTICATOR_VERIFICATION_DATA,
 ];
 
-export const CUSTOM_MESSAGE_KEYS = [
-  OV_OVERRIDE_MESSAGE_KEY.OV_FORCE_FIPS_COMPLIANCE_UPGRAGE_KEY_IOS,
-  OV_OVERRIDE_MESSAGE_KEY.OV_FORCE_FIPS_COMPLIANCE_UPGRAGE_KEY_NON_IOS,
-  OV_OVERRIDE_MESSAGE_KEY.OV_QR_ENROLL_ENABLE_BIOMETRICS_KEY,
-  OV_UV_ENABLE_BIOMETRIC_SERVER_KEY,
-];
-
 export const AUTHENTICATOR_ENROLLMENT_DESCR_KEY_MAP: Record<string, string> = {
   [AUTHENTICATOR_KEY.EMAIL]: 'oie.email.authenticator.description',
```

Comment:
possible to include these keys in i18nTransformer?

Reply:
This is another adhoc element that is created for a specific flow, it provides a description, this isnt something that comes from the API. 

unless you mean to move the constant to the i18n transformer file?

Reply:
maybe a key like `authenticator.enrollment.email.description` in i18n transformer's map, based on the pattern we introduced, we can programatically link elements to proper i18n keys.

Reply:
Can I create a separate jira ticket to do this refactor? It will require a new map, and adding keys to the existing elements that can correspond to the new map. I just don't want this PR to grow even more than it already is 😓 

But I agree it can be even more generalized.